source/Locales/CompactRegular.lagda

```diff
@@ -1098,6 +1191,99 @@ is-spectral-map F G (f , _) =
 
 \end{code}
 
+-- directification-preserves-coherence : (F : Frame 𝓤 𝓥 𝓦)
+--                                     → (ℬ : Fam 𝓦 ⟨ F ⟩)
+--                                     → (σ : closed-under-finite-meets F ℬ holds)
+--                                     → closed-under-finite-meets F (directify F ℬ) holds
+-- directification-preserves-coherence F ℬ c@(τ , σ) = β , γ
+--  where
+--   open PosetReasoning (poset-of F)
+--   open Meets (λ x y → x ≤[ poset-of F ] y) hiding (is-top)
+
+--   β : contains-top F (directify F ℬ) holds
+--   β = ∥∥-rec (holds-is-prop (contains-top F (directify F ℬ))) † τ
+--        where
+--         † : Σ t ꞉ index ℬ , is-top F (ℬ [ t ]) holds
+--           → contains-top F (directify F ℬ) holds
+--         † (t , p) = ∣ (t ∷ []) , transport (λ - → is-top F - holds) (‡ ⁻¹) p ∣
+--          where
+--           ‡ : directify F ℬ [ t ∷ [] ] ＝ ℬ [ t ]
+--           ‡ = ℬ [ t ] ∨[ F ] 𝟎[ F ]  ＝⟨ 𝟎-left-unit-of-∨ F (ℬ [ t ]) ⟩
+--               ℬ [ t ]                ∎
+
+--   γ : closed-under-binary-meets F (directify F ℬ) holds
+--   γ is js = ∥∥-rec₂ ∥∥-is-prop δ (coherence-list F ℬ c is) (coherence-list F ℬ c is)
+--    where
+--     δ : (Σ m ꞉ index ℬ , ℬ [ m ] ＝ finite-meet F ℬ is)
+--       → (Σ n ꞉ index ℬ , ℬ [ n ] ＝ finite-meet F ℬ is)
+--       → ∥ Σ ks ꞉ index (directify F ℬ) ,
+--            ((directify F ℬ [ ks ]) is-glb-of (directify F ℬ [ is ] , directify F ℬ [ js ])) holds ∥
+--     δ (m , μ) (n , ν) = ∥∥-rec ∥∥-is-prop ϵ (σ m n )
+--      where
+--       ϵ : Sigma (index ℬ) (λ k → ((ℬ [ k ]) is-glb-of (ℬ [ m ] , ℬ [ n ])) holds)
+--         → ∥ Sigma
+--              (index (directify F ℬ))
+--              (λ ks → ((directify F ℬ [ ks ]) is-glb-of (directify F ℬ [ is ] , directify F ℬ [ js ])) holds) ∥
+--       ϵ (k , ξ) = ∣ (k ∷ []) , (ζ₁ , ζ₂) , θ ∣
+--        where
+--         ζ₁ : ((directify F ℬ [ k ∷ [] ]) ≤[ poset-of F ] (directify F ℬ [ is ])) holds
+--         ζ₁ = ℬ [ k ] ∨[ F ] 𝟎[ F ]                             ≤⟨ {!!} ⟩
+--              ℬ [ k ]                                           ≤⟨ {!!} ⟩
+--              ℬ [ m ] ∧[ F ] ℬ [ n ]                            ≤⟨ {!!} ⟩
+--              (finite-meet F ℬ is) ∧[ F ] (finite-meet F ℬ js)  ≤⟨ {!!} ⟩
+--              finite-meet F ℬ is                                ≤⟨ {!!} ⟩
+--              directify F ℬ [ is ]                              ■
+
+--         ζ₂ : {!!}
+--         ζ₂ = {!!}
+
+--         θ : {!!}
+--         θ = {!!}
+
+\end{code}
+
+\begin{code}
+
+compact-rel-syntax : (F : Frame 𝓤 𝓥 𝓦) → ⟨ F ⟩ → ⟨ F ⟩ → Ω (𝓤 ⊔ 𝓥 ⊔ 𝓦 ⁺)
+compact-rel-syntax F U V =
+ Ɐ W ∶ ⟨ F ⟩ , is-compact-open F W ⇒ W ≤[ poset-of F ] U ⇒ W ≤[ poset-of F ] V
+
+syntax compact-rel-syntax F U V = U ≤ₖ[ F ] V
+
+spectral-yoneda : (F : Frame 𝓤 𝓥 𝓦) → is-spectral F holds → (U V : ⟨ F ⟩)
```

Comment:
When a type is too long, it is better to put each arrow in a separate line, at the beginning.

Reply:
Done.